packages/treasury/src/vaultManager.js

```diff
@@ -42,7 +44,7 @@ export function makeVaultManager(
 ) {
   const { brand: runBrand } = runMint.getIssuerRecord();
 
-  const shared = {
+  const shared = Far('shared vault something', {
```

Comment:
Shouldn't need a `Far` declaration. The only uses are with spread operator to add these operations to other `Far` objects.  (okay, and one direct use, but it never escapes this file.)

Reply:
correct. fixed